### ISSUE TITLE
Consider a sticky footer

### ISSUE THREAD

**Issue by chrismilway** [plone/plonetheme.barceloneta#36]
Particularly when the toolbar is present it might be a good idea to include a sticky footer. Entirely superficial, I know, but a nice addition if it's straightforward.

http://philipwalton.github.io/solved-by-flexbox/demos/sticky-footer/

This might be a good option because it works with variable height footers and sexy flexbox layout.

**Comment by vikas-parashar** [comment]
Yeah, ideally footer should be on bottom even when the main container has less content. 

![image](https://cloud.githubusercontent.com/assets/5694967/12494969/0ab5155c-c0b3-11e5-8889-21771d2c040e.png)

**Comment by pbauer** [comment]
:+1:

**Comment by thet** [comment]
+1

**Comment by draunger** [comment]
+1

**Comment by Rudd-O** [comment]
This would be hawt.
